dist/index.js

```diff
@@ -66,4 +77,30 @@ github.repos.fork({
   } else {
     console.log('Hey it worked!');
   }
-});
\ No newline at end of file
+});
+
+function packagePath(dep) {
+  var cwd = process.cwd().match(/([^\/\\]+$)/)[1];
+  if (!dep || cwd == dep) {
```

Comment:
Let's default to `===` instead of  `==` due to JavaScript's psychopathy: http://stackoverflow.com/questions/359494/does-it-matter-which-equals-operator-vs-i-use-in-javascript-comparisons